src/coreclr/src/tools/ReadyToRun.SuperIlc/BuildOptions.cs

```diff
@@ -36,11 +36,12 @@ public class BuildOptions
         public FileInfo[] IssuesPath { get; set; }
         public FileInfo R2RDumpPath { get; set; }
         public FileInfo CrossgenResponseFile { get; set; }
-        public DirectoryInfo[] RewriteOldPath { get;set; }
-        public DirectoryInfo[] RewriteNewPath { get;set; }
+        public DirectoryInfo[] RewriteOldPath { get; set; }
+        public DirectoryInfo[] RewriteNewPath { get; set; }
         public bool MeasurePerf { get; set; }
         public string InputFileSearchString { get; set; }
         public string ConfigurationSuffix => (Release ? "-ret.out" : "-chk.out");
+        public string GCStress {get; set; }
```

Comment:
Nit - as you seem to be beautifying spacing, you're missing a space after the opening bracket ;-).

Reply:
The beautifying change was actually made by the VS, but thanks for noticing that I've made it ugly here.